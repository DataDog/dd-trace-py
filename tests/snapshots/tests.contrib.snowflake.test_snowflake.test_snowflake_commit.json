[[
  {
    "name": "snowflake.connection.commit",
    "service": "snowflake",
    "resource": "snowflake.connection.commit",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "",
    "error": 0,
    "meta": {
      "_dd.p.dm": "-0",
      "component": "snowflake",
      "db.application": "PythonConnector",
      "db.name": "mock-db-name",
      "db.schema": "mock-schema-name",
      "db.system": "snowflake",
      "db.user": "mock-user",
      "db.warehouse": "mock-warehouse-name",
      "language": "python",
      "out.host": "mock-account.snowflakecomputing.com",
      "runtime-id": "f82097569d3141b8ba61409269818dec",
      "span.kind": "client"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "network.destination.port": 443,
      "process_id": 79541
    },
    "duration": 1358000,
    "start": 1667243938376321000
  }]]
