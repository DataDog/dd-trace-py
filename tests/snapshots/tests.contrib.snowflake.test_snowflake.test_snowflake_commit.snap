[[{"name" "snowflake.connection.commit"
   "service" "db"
   "resource" "snowflake.connection.commit"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1633556569118781400
   "duration" 3784200
   "meta" {"runtime-id" "f8ec8599814b4c7c8dba46062fbed9a6"
           "out.host" "mock-account.snowflakecomputing.com"
           "db.name" "mock-db-name"
           "db.user" "mock-user"
           "db.application" "PythonConnector"
           "db.schema" "mock-schema-name"
           "db.warehouse" "mock-warehouse-name"}
   "metrics" {"_dd.agent_psr" 1.0
              "_sampling_priority_v1" 1
              "system.pid" 6052
              "out.port" 443
              "_dd.tracer_kr" 1.0}}]]
